instaclustr/resource_cluster.go

```diff
@@ -1235,6 +1240,20 @@ func resourceClusterRead(d *schema.ResourceData, meta interface{}) error {
 	return nil
 }
 
+func deleteAttributesConflictWithDataCentres(d *schema.ResourceData) error {
+	clusterResource := resourceCluster()
+	for key, value := range clusterResource.Schema {
+		for _, conflictsWith := range value.ConflictsWith {
+			if _, exist := d.GetOk(key); exist && conflictsWith == "data_centres" {
+				if err := d.Set(key, value.Type.Zero()); err != nil {
+					return err
```

Comment:
How perspicuous is this error?  If something does go wrong, will folks easily be able to track down the source of the error?

Reply:
I don't believe d.Set here should have an error ever, but if it does the error should be pretty explanatory